server/utils/request.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestInterface } from '../models/requestModel'
```

Comment:
bit nitpicky, can you move /utils folder to /graphql/utils

Reply:
Updated.